docs/prover/cli/options.md

```diff
@@ -647,11 +653,7 @@ Options that help reduce the running time
 
 ### `--method`
 
-**What does it do?**
-Parametric rules will only verify the method with the given signature, instead of all public and external methods of the contract. Note that you will need to wrap the method's signature with quotes, as the shell doesn't interpret parenthesis correctly otherwise.
-
-**When to use it?**
-When you are trying to solve/understand a counterexample of a parametric rule on a specific method.
+See {ref}`--method`
 
 **Example**
```

Comment:
Not sure if this is needed - there is an example in the link, and better to not keep several references which might get out of sync

Reply:
Yah I missed this. Removing it as well.